README.md

```diff
@@ -12,6 +12,18 @@ unified interface.
 
 #### Note: This package is still in development. The README is currently both an active documentation and a development roadmap.
 
+## Installation
+
+Assuming that you already have Julia correctly installed, it suffices to import
+GalacticOptim.jl in the standard way:
+
+```julia
+import Pkg; Pkg.add("GalacticOptim")
+```
+The packages relevant to the basic functionality of GalacticOptim.jl will be imported
+accordingly and, in most cases, you do not have to worry about the manual
+installation of dependencies.
```

Comment:
You might want to mention that we do need to import some of the optimisation backends since they are not reexported. I see you mention it at https://github.com/SciML/GalacticOptim.jl/pull/66/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44 but a quick mention of all the backends. BBO, NLopt, etc. would be good to throw in here

Reply:
Great, thanks for pointing it out!

Reply:
@mkg33 please follow up on this from a new PR now

Reply:
@Vaibhavdixit02 Sure, I've compiled the list of the extra packages and will add it in the new PR.